Spawn-App-iOS-SwiftUI/Views/ProfileView.swift

```diff
@@ -13,55 +13,147 @@ struct ProfileView: View {
     var body: some View {
         NavigationStack {
             ScrollView {
-                VStack(alignment: .leading, spacing: 20) {
+                VStack(alignment: .center, spacing: 20) {
                     // Profile Picture
                     
                     if let profilePicture = appUser.profilePicture {
                         profilePicture
                             .ProfileImageModifier(imageType: .profilePage)
+                            .clipShape(Circle())
```

Comment:
Hey Michael, these lines actually aren't necessary, since they've done in `.ProfileImageModifier()`. Could you edit the `switch-case` in that function directly?

Reply:
![image](https://github.com/user-attachments/assets/a2e6c532-1075-48f4-a546-05b137d8f4d7)